classes/install/Upgrade.inc.php

```diff
@@ -1016,9 +1016,6 @@ public function migrateUserAndAuthorNames()
             $suffix = $row->suffix;
             foreach ($supportedLocales as $siteLocale) {
                 $preferredPublicName = ($salutation != '' ? "${salutation} " : '') . "${firstName} " . ($middleName != '' ? "${middleName} " : '') . $lastName . ($suffix != '' ? ", ${suffix}" : '');
-                if (AppLocale::isLocaleWithFamilyFirst($siteLocale)) {
```

Comment:
I saw this was always false, so I removed. But if users are supposed to configure the locales.xml, then I can bring it back.